src/tools/aml-language-server/handlers.ts

```diff
@@ -0,0 +1,141 @@
+/**
+ * @license
+ * Copyright (c) 2019 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {Message} from 'vscode-jsonrpc';
+import {NotificationMessage} from 'vscode-jsonrpc/lib/messages';
+import {CodeActionParams, Command, CompletionItemKind, CompletionList, Diagnostic, DiagnosticSeverity, DidChangeConfigurationParams, DidChangeTextDocumentParams, DidCloseTextDocumentParams, DidOpenTextDocumentParams, DidSaveTextDocumentParams, DocumentSymbolParams, ExecuteCommandParams, Hover, InsertTextFormat, Location, MarkedString, ParameterInformation, PublishDiagnosticsParams, Range, ReferenceParams, RenameParams, SignatureHelp, SignatureInformation, SymbolInformation, TextDocumentPositionParams, TextDocumentSyncKind, TextEdit, WorkspaceEdit} from 'vscode-languageserver';
+
+import {Dictionary} from '../../runtime/hot.js';
+import {Manifest, ManifestError} from '../../runtime/manifest.js';
+
+import {LspLoader} from './lspLoader.js';
+import * as MessageTypes from './messageTypes.js';
+import {alphaNumerics, AmlServiceContext, jsonrpc, normalizeUri, uri2path} from './util.js';
+
+// tslint:disable-next-line: no-any
+export type Handler = ((params: object, context: AmlServiceContext) => any);
```

Comment:
Seems like declaring params as `object` here doesn't gain us anything - it's not providing useful type info, and we need to recast as `any` quite a lot below. Just make it `any` here?

Reply:
Done